doc/Requirements Specification/Parts/Test Cases.lyx

```diff
@@ -899,18 +1022,22 @@ gls{Beagle}
 
 \end_inset
 
- reacts to different component's interface parameters.
- Tests 
+ works on a system with only the software specified in 
 \begin_inset CommandInset ref
 LatexCommand ref
-reference "/OF30/"
+reference "/Q10/"
 
 \end_inset
 
- and 
+.
+ A new system has to be set up with only these software applications and
+ a reasonable number of the functionality tests have to be run on it (may
+ for example be achieved by running all tests on a continuous integration
```

Comment:
> I think they're optional.

That’s what I think. And they interrupt the flow of reading. So I vote for leaving them out.